pkg/scripts/generate-variables.sh

```diff
@@ -1,24 +1,26 @@
 # This script will generate/override the `./pkg/tfstate/tfstate-locals.tf` and `./terraform-backend.tf`
-SCRIPT_PATH=$(dirname "$0")
-if [ $# -lt 1 ]; then
-  show_help
-fi
-CONFIG_FILE="${1}"
-if [ $# -eq 1 ]; then
-  ROOT_FOLDER="$(pwd)"
-else
-  ROOT_FOLDER="${2}"
-fi
 
 show_help() {
     echo "The terraform config filename for infrastructure is missing."
     echo
-    echo "Usage: generate-variables.sh <config_file> [<path_to_root>]"
+    echo "Usage: generate-variables.sh <config_file>"
     exit 1
 }
 
+if [ $# -lt 1 ]; then
+  show_help
+fi
+CONFIG_FILE="${1}"
+if [ ! -f "${CONFIG_FILE}" ]; then
+  echo "Could not find config file ${CONFIG_FILE}."
+  show_help
+fi
+
+# this script is located in {repo_root_path}/pkg/scripts
+SCRIPT_PATH="$(dirname "$0")"
+ROOT_PATH="${SCRIPT_PATH}/../.."
```

Comment:
I think we should convert all paths to absolute paths - they are easier to read and more robust.

Reply:
This is outdated code. In latest code we have using absolute path:
```
cd "$(dirname "$0")/../.."
ROOT_PATH="$(pwd)"
cd "${CURRENT_PATH}"
SCRIPT_PATH="${ROOT_PATH}/pkg/scripts"
```